statsmodels/datasets/utils.py

```diff
@@ -132,6 +132,21 @@ def _open_cache(cache_path):
         return zlib.decompress(zf.read())
 
 
+# Sourced from Holoviews:
+# https://github.com/holoviz/holoviews/blob/74fda3fb832257357c48eda1be0a73eb523966a7/holoviews/pyodide.py#L78-L81
+# License: BSD-3-Clause
+def _in_jupyterlite() -> bool:
+    import sys
+    if "pyodide" in sys.modules:
+        import js
+        return (
+            hasattr(js, "_JUPYTERLAB") or
```

Comment:
Which is these tests is the most robust?  I feel like it is the `not hasattr(` one, which should probably be first if so.

Reply:
🤔 yes. I'll check if there was a specific reason why the order exists like this in Holoviews.

Reply:
I changed the order of the checks via 695a4921a4d974e33bce5baba3439b28af51cc53. Please let me know if this addresses this point. Thanks!